src/CloudFunctions/Storage.js

```diff
@@ -0,0 +1,60 @@
+import firebase from "firebase";
```

Comment:
Should this be something like:
```suggestion
import firebase from "firebase/app";
import "firebase/storage";
```

to avoid the firebase warnings (smiliar to stuff in #110?)